### ISSUE TITLE
Wrong user name or password

### ISSUE THREAD

**Issue by ipa64** [Kilhog/Alfred-Workflows#2]
Bonjour,
J'essaye de faire fonctionner ce workflow, sans succès.
J'obtiens toujours le message:
Wrong username or password.
Bien entendu j'ai tout bien contrôlé

Peut être est-ce dû au changement de domaine de T411?
Merci pour votre travail

**Comment by Kilhog** [comment]
Oui ça doit être ça, je change le domaine tout à l'heure ;) je te tiens au courant

**Comment by ipa64** [comment]
Effectivement, j'ai refais le package avec la dernière modification que quelqu'un à fait sur le fichier global.php et ça fonctionne.
Merci

**Comment by ipa64** [comment]
Encore une question, mon client torrent tourne sur mon NAS Synology, il y a un plugin dans Safari qui lorsque clique droite sur "Télécharger" je choisi "Télécharger avec diskstation" et hop
J'aimerai pouvoir reproduire cela plutôt que de télécharger le torrent. Tu as une idée comment réaliser cela? Merci!

**Comment by elephantjupo** [comment]
Bonsoir,
même problème pour moi ! Comment puis-je refaire le package ?

**Comment by Kilhog** [comment]
Bonsoir, 
Alors moi j'ai complètement abandonné le truc (j'ai même plus Alfred pour tester).
Si tu veux tu peux importer le fichier Alfred-Workflows/t411-Workflow/T411.alfredworkflow et commencer à le modifier dans les préférences du workflow dans Alfred, dans mes souvenirs.

**Comment by elephantjupo** [comment]
Justement, je ne vois pas où modifier le domaine dans les préférences d'Alfred... Mais pas grave.
